base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: rgba(200, 200, 200, 0.1);
+  width: 400px;
+  height: 75px;
+
+  transition-duration: 5s;
+}
+.mainArea {
+  width: auto;
```

Comment:
`auto` is the default value of `width`, so you don't need to set it

Reply:
Thx, i was having problems with the allignment.
Good to know.

width auto (removed since default)